gradle.properties

```diff
@@ -1,4 +1,4 @@
-version=29.28.0
+version=29.28.1
```

Comment:
so you will be bumping the minor version later? otherwise, new version won't be resolved transitively.

Reply:
Fixed, bumping minor version